CHANGELOG.md

```diff
@@ -1,22 +1,99 @@
+    - KeepAlive Timout log level changed to debug (Arnulfo Solís)
+    - Pin pytest to 3.3.2 because of pytest-dev/pytest#3170 (Maksim Aniskenov)
+    - Install Python 3.5 and 3.6 on docker container for tests (Shahin Azad)
+    - Add support for blueprint groups and nesting (Elias Tarhini)
+    - Remove uvloop for windows setup (Aleksandr Kurlov)
+    - Auto Reload (Yaser Amari)
+    - Documentation updates/fixups (multiple contributors)
+
+  - Fixes:
+    - Fix: auto_reload in Linux (Ashley Sommer)
+    - Fix: broken tests for aiohttp >= 3.3.0 (Ashley Sommer)
+    - Fix: disable auto_reload by default on windows (abuckenheimer)
+    - Fix (1143): Turn off access log with gunicorn (hqy)
+    - Fix (1268): Support status code for file response (Cosmo Borsky)
+    - Fix (1266): Add content_type flag to Sanic.static (Cosmo Borsky)
+    - Fix: subprotocols parameter missing from add_websocket_route (ciscorn)
+    - Fix (1242): Responses for CI header (yunstanford)
+    - Fix (1237): add version constraint for websockets (yunstanford)
+    - Fix (1231): memory leak - always release resource (Phillip Xu)
+    - Fix (1221): make request truthy if transport exists (Raphael Deem)
+    - Fix failing tests for aiohttp>=3.1.0 (Ashley Sommer)
+    - Fix try_everything examples (PyManiacGR, kot83)
+    - Fix (1158): default to auto_reload in debug mode (Raphael Deem)
+    - Fix (1136): ErrorHandler.response handler call too restrictive (Julien Castiaux)
+    - Fix: raw requires bytes-like object (cloudship)
+    - Fix (1120): passing a list in to a route decorator's host arg (Timothy Ebiuwhe)
+    - Fix: Bug in multipart/form-data parser (DirkGuijt)
+    - Fix: Exception for missing parameter when value is null (NyanKiyoshi)
+    - Fix: Parameter check (Howie Hu)
+    - Fix (1089): Routing issue with named parameters and different methods (yunstanford)
+    - Fix (1085): Signal handling in multi-worker mode (yunstanford)
+    - Fix: single quote in readme.rst (Cosven)
+    - Fix: method typos (Dmitry Dygalo)
+    - Fix: log_response correct output for ip and port (Wibowo Arindrarto)
+    - Fix (1042): Exception Handling (Raphael Deem)
+    - Fix: Chinese URIs (Howie Hu)
+    - Fix (1079): timeout bug when self.transport is None (Raphael Deem)
+    - Fix (1074): fix strict_slashes when route has slash (Raphael Deem)
+    - Fix (1050): add samesite cookie to cookie keys (Raphael Deem)
+    - Fix (1065): allow add_task after server starts (Raphael Deem)
+    - Fix (1061): double quotes in unauthorized exception (Raphael Deem)
+    - Fix (1062): inject the app in add_task method (Raphael Deem)
+    - Fix: update environment.yml for readthedocs (Eli Uriegas)
+    - Fix: Cancel request task when response timeout is triggered (Jeong YunWon)
+    - Fix (1052): Method not allowed response for RFC7231 compliance (Raphael Deem)
+    - Fix: IPv6 Address and Socket Data Format (Dan Palmer)
+
+Note: Changelog was unmaintained between 0.1 and 0.7
```

Comment:
FYI: looks like the changelog was maintained in the [GitHub releases](https://github.com/huge-success/sanic/releases) between these versions. It'd be great if we could pull those into this doc.

Reply:
Given that the original maintainers chose not to for some reason, I don't know that we need to go back and step on toes.